packages/autoloader/src/autoload.php

```diff
@@ -72,9 +72,20 @@ function autoloader( $class_name ) {
 		global $jetpack_packages_classes;
 
 		if ( isset( $jetpack_packages_classes[ $class_name ] ) ) {
-
-			if ( defined( WP_DEBUG ) && WP_DEBUG ) {
-				if ( function_exists( 'did_action' ) && ! did_action( 'plugins_loaded' ) ) {
+			if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
+				// TODO ideally we shouldn't skip any of these, see: https://github.com/Automattic/jetpack/pull/12646
+				$ignore = in_array(
+					$class_name,
+					array(
+						'Automattic\Jetpack\JITM\Manager',
+						'Automattic\Jetpack\Connection\Manager',
+						'Automattic\Jetpack\Connection\Manager_Interface',
+						'Jetpack_Options',
+						'Jetpack_Sync_Main',
+						'Automattic\Jetpack\Constants',
+					)
```

Comment:
Let's use strict `in_array`.

```suggestion
					),
					true
```